cognee/tests/test_path_config.py

```
@@ -0,0 +1,114 @@
+    assert result == str(Path(abs_path).resolve())
+    
+    # Test with relative path (should convert to absolute)
+    rel_path = "data/vector.db"
+    result = ensure_absolute_path(rel_path, base_path=system_root, allow_relative=True)
+    expected = str((Path(system_root) / rel_path).resolve())
+    assert result == expected
+    
+    # Test with relative base_path (should fail)
+    try:
+        ensure_absolute_path(rel_path, base_path="relative/base", allow_relative=True)
+        assert False, "Should fail when base_path is relative"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)